docs/java/features/odata/use-vdm.md

```diff
@@ -0,0 +1,213 @@
+        .filter(Person.EMAILS.contains(Collections.singletonList("ASD")))
+        .execute(destination);
+    ```
+  The above query also shows how to perform nested operations on navigation properties `Person.TRIPS` and `Person.FRIENDS`.
+  This is a new functionality introduced as part of supporting OData v4. In addition we aslo see new type-sensitive filter expressions on entity fields.
+
+### Handling and debugging failures
+
+Sometimes requests fail and the SDK provides a flexible way to deal with such failures on multiple levels. All `execute` methods may throw a runtime exception (extending) `ODataException`. This will always contain the request which was (attempted to be) sent out as well as the cause of the exception. To handle all kind of failures consider the following code:
+
+```java
+try { ... }
+ catch( final ODataException e ) {
+    logger.error("Failed to execute OData query.", e);
+    ODataQueryGeneric query = e.getQuery();
+    logger.debug("The following query failed: {}", query);
+    // do something else
+}
+```
+
+This handling is very generic and does not really give a lot of information, except for the request that was (to be) sent out. For more specific information there are dedicated exceptions. Please tend to the documentation of `ODataException` for all the exception types. The following code presents one example which deals with the case that the OData service responded with an HTTP status code indicating an error:
+
+ ```java
+try { ... }
+catch( final ODataServiceException e ) {
+    // ODataQueryGeneric query = e.getQuery();
+    int httpCode = e.getHttpCode();
+    Header[] httpHeaders = e.getHttpHeaders():
+    // react based on the response code
+ }
+ ```
+
+Finally, the OData service might also include an OData error in the payload of the response:
+
+```java
+    try { ... }
+    catch( ODataErrorResponseException e ) {
+      // ODataQueryGeneric query = e.getQuery();
+      // int httpCode = e.getHttpCode();
+      // Header[] httpHeaders = e.getHttpHeaders():
+      String oDataCode = e.getError().getODataCode();
+      String oDataMessage = e.getError().getODataMessage();
+      String target = e.getError().getTarget();
+      List<ODataServiceError> details = e.getError().getDetails();
+      Map<String, Object> innerError = e.getError().getInnerError();
+    }
+```
+
+You can also list multiple catch clauses to cover different levels or cases that might occur, e.g.:
+
+```java
+try { ... }
+catch( ODataErrorResponseException e ) {
+    // ...
+} catch( ODataRequestException e ) {
+    // ...
+} catch( ODataDeserializationException e ) {
+    // ...
+}
+```
+
+Note that instead of applying `try/catch` you can also make use of `tryExecute` on the request builders.
+
+
+## The following features are not yet supported in the implementation of the OData v4 standard:
+* Error handling for failed OData requests
+* Complex Type properties are not available in query builders
```

Comment:
The title and content of this list don't really go together... 
'Complex Type properties are not available in query builders' is not the feature that is missing. 'Usage of Complex Type properties in query builders' is.